app/src/main/java/com/eveningoutpost/dexdrip/Models/Bubble.java

```diff
@@ -31,14 +31,25 @@ public static boolean isBubble() {
         if (activeBluetoothDevice == null || activeBluetoothDevice.name == null) {
             return false;
         }
-        return activeBluetoothDevice.name.contentEquals("Bubble");
+        return activeBluetoothDevice.name.contains("Bubble");
```

Comment:
Maybe, `startsWith` is more appropriate here.

Reply:
Thank you!